app/controllers/listings_controller.rb

```diff
@@ -272,10 +273,19 @@ def edit
     shape = get_shape(@listing.listing_shape_id)
     process = get_transaction_process(community_id: @current_community.id, transaction_process_id: shape[:transaction_process_id])
     unit_options = ListingViewUtils.unit_options(shape[:units], unit_from_listing(@listing))
+    shipping_price_additional =
+      if @listing.shipping_price_additional
```

Comment:
I'd prefer, if these assignments with conditionals would be in their own methods. Makes 'em easier to read.